pathfinderlib/main.py

```diff
@@ -2,22 +2,60 @@
 import os
 import tempfile
 import argparse
+import inspect
 import json
+import re
+import pandas
+from Bio import SeqIO
 
 from pathfinderlib.version import __version__
-#path to humann
 
-default_config ={
-    "humann_fp": "/home/ashwini/ash/other_softwares/humann/",
-    "kegg_fp": "/home/ashwini/ash/kegg/kegg"
-    }
-        
-                                
-class Humann(object):
-    def __init__(self, config):
-        self.config = config
+def get_config(user_config_file):
+    config ={
+        "humann_fp": "/home/ashwini/ash/other_softwares/humann/",
+        "kegg_fp": "/home/ashwini/ash/kegg/kegg",
```

Comment:
Again, uncertain what to do about the default here, but good to have something not user-specific.